theme.toml

```diff
@@ -5,7 +5,7 @@ description = "A minimalistic dark responsive theme."
 homepage = "https://github.com/gyorb/hugo-dusk"
 tags = ["blog", "responsive", "google analytics", "disqus", "syntax highlighting", "custom themes", "minimal", "minimalist", "clean", "dark"]
 features = ["blog"]
-min_version = 0.55
+min_version = 0.57
```

Comment:
The min_version should be a string also Hugo now uses semver versions, so please change this to:
`min_version = "0.57.0"